test/utils/initMatchers.ts

```diff
@@ -106,12 +106,15 @@ chai.use((chaiAPI, utils) => {
 
     this.assert(
       virutallyFocusedElementId === id,
```

Comment:
Apparently I can’t spell either 😂

Reply:
> eps1lon wants to merge 2 commits into mui-org:next from eps1lon:test/**virtually**-focused-error-message

What you mean "either"? :stuck_out_tongue:

Reply:
😂 It was more I can’t write test matchers or spell 😉